Migration-Guide.md

```diff
@@ -63,8 +60,106 @@ Also, the new 1.x validator will honor the `--errors-only` and `--summary-only`
 the v0.x pre-release validator did not.
 More details related to validator output can be found [here](README.md#validator-output).
 
+## API
+The v0.x pre-release validator supported an API to enable users to invoke the validator programmatically
+from a javascript application.
+This API is no longer supported in the v1.x validator; however, if there is a need to enable the
+validator to be invoked programmatically from the user's application, then it could be considered
+for a future release.
```

Comment:
Let's also link to [Spectral's documentation on programmatic validation](https://meta.stoplight.io/docs/spectral/eb68e7afd463e-spectral-in-java-script). Our current recommendation can be that users should use that with our ruleset.